README.md

```diff
@@ -53,6 +53,16 @@ It is very useful to ignore XXX* fields in protobuf generated messages. The go p
     	--gotag_out=xxx="graphql+\"-\" bson+\"-\"":. example/example.proto
 ```
 
+## Output to a directory other than the current directory
```

Comment:
Can you change this to an issue experienced with prototool as not a lot (including me despite using prototool) are not facing the issue. If you can help me replicate I'll be happy to add in the situation

Reply:
Look at the comments I added to Issue #9   It gives extremely simple reproduction steps and the cause.

Reply:
Also once you merge, can you tag so I can use a go mod replace?

Reply:
Are there any plans to still merge this? I would like to use this as well :-)